yt/__init__.py

```diff
@@ -7,13 +7,6 @@
 * Contribute: https://github.com/yt-project/yt
 
 """
-import sys
-
-if sys.version_info[0] < 3:
-    raise Exception(
-        "Python 2 is no longer supported.  Please install Python 3 for use with yt."
-    )
-
```

Comment:
I'm removing this check because it is impossible to trigger: yt _cannot be installed_ with Python 2 (or any version <3.6), as the requirement is already specified in `setup.cfg`, so the failure should and will happen at compile time rather than runtime.